slow_odgi/README.md

```diff
@@ -0,0 +1,350 @@
+### Overview
```

Comment:
I'd recommend starting with `# slow-odgi` so the document itself has a nice title.